.cargo/config

```diff
@@ -0,0 +1,10 @@
+[target.xtensa-esp32-none-elf]
+runner = "xtensa-esp32-elf-gdb -q -x xtensa.gdb"
+
+[build]
+rustflags = [
+  "-C", "link-arg=-nostartfiles",
+  "-C", "debuginfo=0", # not 2
```

Comment:
This is no longer required, the new llvm fork supports debug info.

Reply:
Just for clarification, am I removing the entire `rustflags` array, its first 2 elements, or just the `debuginfo` line?

Reply:
@MabezDev Which exact llvm fork are you using? If I use https://github.com/espressif/llvm-project/tree/xtensa_release_9.0.1 I still get CFI not supported errors.

Reply:
@arjanmels I'm using that fork; it should work. Note that if you're trying to compile .S assembly files, it will be passed to the gcc assembler at the moment, which will produce these errors. Inline assembly works though.

Reply:
Strange, not working for me. I recompiled llvm and rust, but no luck, must be missing something.

This is what I get if I check my version (afraid it may not be super helpfull:

$RUSTC --version --verbose
rustc 1.44.0-dev
binary: rustc
commit-hash: unknown
commit-date: unknown
host: x86_64-unknown-linux-gnu
release: 1.44.0-dev
LLVM version: 9.0

Reply:
That is weird, so you can't create debug info at all? I've defintely got it working on my machine: https://github.com/espressif/llvm-project/issues/10#issuecomment-575854084

Ive been pretty tied up this week, but today I've been revisiting the rust fork, looking to improve the ABI calls, and use the built in llvm module, which *might* fix your issue.

Reply:
No debug info not working for me. If I understand your comment for https://github.com/espressif/llvm-project/issues/10#issuecomment-575854084 correct you are working with a local rebase, so theoretically could be something different?

More likely that I did something wrong perhaps, but I don't know what. Anyway I am managing ok without the debugger at the moment as I have the serial output up and running, but debugger would be nice of course.

Reply:
That rebase was temporary, the current default branch of the fork has the correct changes in it.